http/get_simple/ruby/client/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# HTTP GET Arrow Data: Simple Ruby Client Example
+
+This directory contains a minimal example of an HTTP client implemented in Ruby.
+
+The client:
+
+1. Sends an HTTP GET request to a server.
+2. Receives an HTTP 200 response from the server, converts the response to a RecordBatchReader, and consumes it.
+3. Sends a second HTTP GET request to a server.
+with the response body containing an Arrow IPC stream of record batches.
+4. Streams the response, creating and printing record batches as it goes.
```

Comment:
Updated this to match the current example. We should keep this simple; it's just meant to describe what happens at the highest level, not exactly how it's wired up.
```suggestion
2. Receives an HTTP 200 response from the server, with the response body containing an Arrow IPC stream of record batches.
3. Adds the record batches to an Array as they are received.
```